docs/respondent_management_to_electronic_questionnaire.rst

```diff
@@ -25,6 +25,12 @@ Schema Definition
     The trading as name for a responding unit. Temporary until wider refactor.
   eq_id
     The eQ questionnaire instance id.
+  case_id
+    The case UUID used to identify a single instance of a survey collection for a respondent
+  case_ref
+    The case reference identified by the above UUID (e.g. "1000000000000001")
+  account_service_url
+    The url of the account service (i.e. rrm or ras) to send the user back to once they've completed the EQ survey
```

Comment:
I think we should be less specific about this - something like "The url of the account service (i.e. rrm or ras) used to launch the survey". 

We only use this currently on the save and sign out functionality - the link doesn't appear on the submission success page.

Reply:
Done